### PR TITLE
Feature/26630 prepared check sync prerequisites 

### PR DESCRIPTION
**Pull request by GeorgeChatzigiannis**
Added necessary config, migration changes and queues for prerequisites

### CODE REVIEW

File: functions/prepaped-check-sync/message-schema/message.v1.json [hunk 1]
```diff
@@ -0,0 +1,78 @@
+{
```
Comment (GuyHarwood) [inline functions/prepaped-check-sync/message-schema/message.v1.json]: is it the responsibility of the message sender to provide the updated payload?  or should it just indicate the checkCode of the check that needs resyncing?  the latter seems feels like a better choice

**Comment by GuyHarwood** [comment]
@GeorgeChatzigiannis added CI label, as you are adding DB changes that need regression testing

File: functions/prepaped-check-sync/message-schema/message.v1.json (hunk 1 above)
Comment (GeorgeChatzigiannis) [reply functions/prepaped-check-sync/message-schema/message.v1.json]: @GuyHarwood I would think the prepared-check-sync function will fetch the relevant `PartitionKey`, `RowKey `,  and `config` then update the latter with the new settings and push to table storage?
In that case the message would not need the checkCode but only these 3 properties that I have highlighted.

Thoughts? 🤔

File: admin/data/sql/migrations/183.do.index-pupil-urlslug-check-checkcode.sql [hunk 2]
```diff
@@ -0,0 +1,2 @@
+CREATE INDEX check_checkCode_index ON [mtc_admin].[check] (checkCode)
```
[the remark this answers is not in the document]
Comment (GeorgeChatzigiannis) [reply admin/data/sql/migrations/183.do.index-pupil-urlslug-check-checkcode.sql]: @jon-shipley @GuyHarwood updated.

**Pull request merged**